CODE_OF_CONDUCT.md

```diff
@@ -117,11 +117,13 @@ accordingly.
 
 ## Attribution
 
-This Code of Conduct is adapted from the [Contributor Covenant][homepage],
```

Comment:
Good catch! Would you mind updating it here instead: https://github.com/adafruit/cookiecutter-adafruit-circuitpython/blob/master/%7B%7B%20cookiecutter.library_name%20%7D%7D/CODE_OF_CONDUCT.md ? We copy this file into many repos and will need to do a broader automated patch run to fix it everywhere.

Reply:
Thanks! And sure, can do. 🙂

Reply:
Done! https://github.com/adafruit/cookiecutter-adafruit-circuitpython/pull/81